tests/components/application-connector/docs/compass-runtime-agnet-tests.md

```diff
@@ -99,6 +106,30 @@ To run the tests without removing all the created resources afterwards, run them
 
 2. Once you've finished debugging, run:
 
+   ``` shell
+   make clean-test-compass-runtime-agent-test
+   ```
+### Debugging in the IDE
+
+To run the test in your IDE you must perform the following steps.
+
+1. To prepare the cluster for debugging, run the test without cleanup:
+
+   ``` shell
+   make test-compass-runtime-agent-debug
+   ```
+
+2. Before starting debugger in your IDE export the following environment variables:
+- KUBECONFIG={Your cluster kubeconfig}
```

Comment:
add 3 blank spaces at the beginning of this line and the following lines of the list, so they're indented correctly below the `2. ` step.